frontend/public/index.html

```diff
@@ -9,7 +9,7 @@
     <!-- Any updates to this meta tag must be reflected in the applications nginx.conf.template for these changes to take effect in openshift-->
     <meta
       http-equiv="Content-Security-Policy"
-      content="default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca http://localhost:8080; connect-src 'self' https://*.oidc.gov.bc.ca/ http://localhost:8080/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'unsafe-hashes' 'unsafe-inline';"
+      content="default-src 'self'; frame-ancestors 'self'; img-src 'self' data: https://openmaps.gov.bc.ca/ https://*.tile.openstreetmap.org/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca http://localhost:8080; connect-src 'self' https://*.oidc.gov.bc.ca/ http://localhost:8080/ https://openmaps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'unsafe-hashes' 'unsafe-inline'; script-src 'self' 'sha256-yt+SNVxRkIi6H6yb7ndFuZM1esMX9esg3UpRHaTsyVk=';"
```

Comment:
Is this going to have a new `sha256` every release?

Reply:
no, this shouldn't change unless the underlying external scripts change. Those shouldn't change unless we bump the versions of external dependencies.